roadmaps/2022roadmap.md

```diff
@@ -0,0 +1,91 @@
+
+### Building an OpenMDAO training course
+
+While no substitution for a training course, the documentation update achieved by the team in 2021 using JupyterBook
+is a signficant advance in interactivity.
+The ability for users to test out code in the documentation online via Binder or Google Colab without installing anything locally significantly lowered the barrier to entry for OpenMDAO.
+However, this means that building a signficant training course is still on the todo list for 2022.
+The focus on various Jupyter notebook based tools should help this effort signifcantly, however.
+
+### General performance improvments
+
+We've made strides on improving performance.  In particular, total derivative coloring is substantially cheaper.
+
+Metamodel interpolation has been a consistent bottleneck, largely due to allowing a variable number of dimensions for each interpolant.
+By developing a series of fixed-dimension interpolants for structured data, we've been able to achieve considerably faster interpolation and differentiation.
+
+While not on the 2021 roadmap, the new function wrapping capability is a big deal that lowers the barrier to entry for OpenMDAO.
+Users no longer need to rewrite their existing functions as OpenMDAO components, and can put the `omf` function wrappers around them instead.
+This is an avenue to the use of automatic differentiation via Google `jax`.
+
+### No Copy Transfers
+
+We worked on No Copy transfers for several weeks and weren't able to work all of the kinks out of the system.
+To date this remains a goal.
+
+# 2022 Focus Areas
+
+## Training
+
+With our advancements in Jupyter notebook-based tools and documentation, the next step is to develop and interactive training course with these as the foundation.
+
+### Goal:
+
+Provide a series of Jupyter notebooks, separate from the documentation, which entail a training course on optimization.
+Starting from basics of single disicipline optimization, work up in complexity to systems involving implicit behavior, multi-point optimization, and cover how to identify degenerate cases.
+
+The training course should be paralleled by videos providing further depth than the example notebooks.
+Teach users to get a feel for how to debug models when things aren't working.
+
+## Aiming for Larger Components
+
+### Goal:
+
+There are two ways in which we can focus on achieving larger monolithic components.
+We've experimented with the use of subproblems and by keeping a lot of their internal data out of the vectors of the outer problem, we've noticed signficant performance increases.
+We will document the use of subproblems.  If it becomes a common pattern, then some sort of official API for them might be warranted, but it's too early to make that call.
+
+Developing training and documentation of automatic differentiation in the form of Google's `jax` as well as `sympy` and its lambdify feature.
```

Comment:
Maybe number the 2 ways.

The sentence starting with "Developing..." is not a sentence. If we just want bullet points for the two ways, then the first bullet point needs to be rewritten to not be a sentence, and to be structured parallel to this one.